README.md

```diff
@@ -9,124 +9,136 @@ The following repository contains:
 - Golang server for processing the data.
 - React client to display game information.
 
+## Table of contents
+
+- [Overview](#overview)
+- [Table of content](#table-of-content)
+- [Features](#features)
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Launching](#launching)
+  - [Launching the server and web application](#launching-the-server-and-web-application)
+  - [Launching the ball-tracking software](#launching-the-ball-tracking-software)
+- [Contact Information](#contact-information)
+
 ## Features
 
-- Keeping track of the score.
-- Manually adjust the score.
+- Keeping track of the score
+- Manually adjusting the score
 
 ## Prerequisites
 
-- [python](https://www.python.org/downloads/macos/) in version 3.10.5 or higher
-- [node](https://nodejs.org/en/download/) in version 16.16.0 or higher
-- npm in version 8.11.0 or higher
-- [go](https://go.dev/dl/) in version 1.18.3 or higher
-- [SmartKickersAI](https://github.com/HackYourCareer/SmartKickersAI)
-- [ximea camera drivers](https://www.ximea.com/support/wiki/apis/ximea_macos_software_package) (macOS)
-- Docker API [for example Docker Desktop](https://docs.docker.com/desktop/install/mac-install/) (macOS)
-- opencv-python
+Get the following:
 
-  ```bash
-  pip3 install opencv-python
-  ```
+1. [Git](https://git-scm.com/downloads)
+2. [Python](https://www.python.org/downloads/) in version 3.10.5 or higher
+3. [Node.js](https://nodejs.org/en/download/) in version 16.16.0 or higher
+4. Access to [SmartKickersAI](https://github.com/HackYourCareer/SmartKickersAI)
 
-  For **brew** users:
+   > **NOTE:** To get access to the SmartKickersAI repository, [contact](#contact-information) Team Beavers.
 
-  ```bash
-  brew install opencv
-  ```
+5. [Ximea camera drivers](https://www.ximea.com/support/wiki/apis/)
+6. Docker daemon for example Docker Desktop:
+   - [MacOS](https://docs.docker.com/desktop/install/mac-install/)
+   - [Windows](https://docs.docker.com/desktop/install/windows-install/)
+   - [Linux](https://docs.docker.com/desktop/install/linux-install/)
+7. [NumPy](https://numpy.org/install/)
 
-- numpy
+> **TIP:** If you're a macOS user, you can also use `brew` to install NumPy:
 
-  ```bash
-  pip3 install numpy
-  ```
+```bash
+brew install numpy
+```
 
-  For **brew** users:
+8. [OpenCV-Python](https://pypi.org/project/opencv-python/)
 
-  ```bash
-  brew install numpy
-  ```
+9. [Imutils](https://pypi.org/project/imutils/)
 
-- imutils
+Check if you have installed all of the [prerequisites](https://github.com/HackYourCareer/SmartKickers#prerequisites).
```

Comment:
What's the point of this line? It refers you to the section you're just reading now. (I suspect it's a leftover from when you had two sections - Prerequisites and Dependencies, if I remember correctly). 
I'd remove it. 

```suggestion
```